src/main/java/frc/robot/RobotContainer.java

```diff
@@ -92,9 +94,9 @@
 
   private Trigger testButton;
 
-  private final SlewRateLimiter m_xspeedLimiter = new SlewRateLimiter(3);
-  private final SlewRateLimiter m_yspeedLimiter = new SlewRateLimiter(3);
-  private final SlewRateLimiter m_rotLimiter = new SlewRateLimiter(3);
+  // private final SlewRateLimiter m_xspeedLimiter = new SlewRateLimiter(3);
+  // private final SlewRateLimiter m_yspeedLimiter = new SlewRateLimiter(3);
+  // private final SlewRateLimiter m_rotLimiter = new SlewRateLimiter(3);
```

Comment:
- [ ] Remove these entirely, simply uncommenting won't give us the functionality back